lib/recording/RecordingEntrySession.js

```diff
@@ -100,6 +119,12 @@ class RecordingEntrySession {
     /*
      In order to check that all chunks in live directory exist in recording, we check first that all chunks in recording are in json
      */
+    updateEntry(){
```

Comment:
let duration = this.getDuration()
if (force || duration>this.lastUpdateDuration+ recordingUpdateEntryTimeIntervalMs) {
       this.lastUpdateDuration = duration
         return backendClient.updateEntry(this.recordedEntryId, duration).then(()=>{
             this.logger.info("Successfully update recorded with duration %s", duration)
         })
}